scripts/generate.v3.ts

```diff
@@ -2,13 +2,16 @@
 import { __unsafe__use__it__at__your__own__risks__wrapDocument, v3 } from "@govtechsg/open-attestation";
```

Comment:
do we need to commit the generated files ?

Reply:
yes pls, it takes a while to generate all of them, its just to generate test fixtures infrequently...